blueos_3_ppc64le_ib/compilers.yaml

```diff
@@ -117,7 +117,7 @@ compilers:
     environment: {}
     extra_rpaths: []
 - compiler:
-    spec: clang@ibm.9.0.0
+    spec: clang@9.0.0.ibm
     paths:
       cc: /usr/tce/packages/clang/clang-ibm-2019.10.03/bin/clang
```

Comment:
Last question... this is using the 2019.10.3 clang version in the path.. should the spec name be updated too then?

Reply:
Yea I remember talking about this earlier in the week with someone, but they mentioned getting rid of all the clang@9 because it was so old.. in this case, it would be a pseudo clang@9 and if they just did a search for it, they'd actually be removing a much newer/preferred clang

Reply:
Fore the records, it is totally fine setting the spec as `clang@9.0.0.ibm.2019.10.03`for example.

Reply:
But it’s a little long to type in an install spec :)